zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/cache/ElasticSearchProcessCacheLoader.java

```diff
@@ -21,10 +25,13 @@ public class ElasticSearchProcessCacheLoader implements CacheLoader<Long, Cached
   private final ElasticsearchClient client;
   private final String processIndexName;
 
+  private final XMLUtil xmlUtil;
+
   public ElasticSearchProcessCacheLoader(
-      final ElasticsearchClient client, final String processIndexName) {
+      final ElasticsearchClient client, final String processIndexName, final XMLUtil xmlUtil) {
```

Comment:
🔧 Could we either introduce a functional interface or use a BiFunction to not depend on the XMlUtil directly?

Like define a interact where you pass in the XML bytes and the processId and get back the List of Call activities ? 

I think this makes it easier to test, and reduces unnecessary dependencies.